languages/elixir/reference/exercise-concepts/secret-handshake.md

```diff
@@ -0,0 +1,53 @@
+# Concepts for secret-handshake
+
+## Optimal solution
+```elixir
+defmodule SecretHandshake do
+  use Bitwise
+
+  @doc """
+  Determine the actions of a secret handshake based on the binary
+  representation of the given `code`.
+
+  If the following bits are set, include the corresponding action in your list
+  of commands, in order from lowest to highest.
+
+  1 = wink
+  10 = double blink
+  100 = close your eyes
+  1000 = jump
+
+  10000 = Reverse the order of the operations in the secret handshake
+  """
+  @spec commands(code :: integer) :: list(String.t())
+  def commands(code) do
+    []
+    |> do_commands(0x08 == (code &&& 0x08), "jump")
+    |> do_commands(0x04 == (code &&& 0x04), "close your eyes")
+    |> do_commands(0x02 == (code &&& 0x02), "double blink")
+    |> do_commands(0x01 == (code &&& 0x01), "wink")
+    |> do_commands(0x10 == (code &&& 0x10), &Enum.reverse/1)
+  end
+
+  defp do_commands(acc, do?, todo)
```

Comment:
I have almost no Elixir experience, but this overloading functionality, is that referred to as multi-clause functions? If so, I'd add it as a Concept too.

Reply:
They are, we should add this, I'll suggest the change in #855 as well

Reply:
👍